src/EnergyPlus/PoweredInductionUnits.cc

```diff
@@ -1608,6 +1593,9 @@ namespace PoweredInductionUnits {
 		QZnReq = ZoneSysEnergyDemand( ZoneNum ).RemainingOutputRequired;
 		QToHeatSetPt = ZoneSysEnergyDemand( ZoneNum ).RemainingOutputReqToHeatSP;
 		CpAirZn = PsyCpAirFnWTdb( Node( ZoneNode ).HumRat, Node( ZoneNode ).Temp );
+		// Initialize local fan flags to global system flags
+		PIUTurnFansOn = ( DataHVACGlobals::TurnFansOn || DataHVACGlobals::TurnZoneFansOnlyOn );
+		PIUTurnFansOff = DataHVACGlobals::TurnFansOff;
```

Comment:
Yeah, the night cycle manager doesn't use TurnFansOff, but other availability managers might.  So, for completeness it seemed like the right thing to do to include that and pass the pair of flags, instead of just passing PIUTurnFansOn to the fan simulation.  I could probably remove it and not see any changes.  I need to look more closely at how the fan simulation uses these.